src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -1018,32 +1022,37 @@ export function submitAppointmentOrRequest(history) {
 
       try {
         let requestData;
-        if (isCommunityCare) {
+        if (featureVAOSServiceRequests && isCommunityCare) {
+          requestBody = transformFormToVAOSCCRequest(getState());
+          requestData = await createAppointment({ appointment: requestBody });
```

Comment:
There's only one appointment endpoint, for both requests and regular appointments, so I called it createAppointment and eventually every appointment/request submission will go through that.